package.json

```diff
@@ -45,10 +48,14 @@
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-sonarjs": "^0.24.0",
     "husky": "^9.0.11",
+    "jest": "29.7.0",
+    "jest-junit": "16.0.0",
     "knip": "^5.0.1",
     "lint-staged": "^15.2.2",
     "npm-run-all": "^4.1.5",
     "prettier": "^3.2.5",
+    "ts-jest": "29.1.2",
+    "ts-node": "10.9.2",
```

Comment:
Could tsx be used possibly or is this required for jest to use a ts config

Reply:
It seems to be an open issue still:
https://github.com/jestjs/jest/issues/13143